.github/workflows/pr-labels.yml

```
@@ -0,0 +1,160 @@
+  pull-requests: write
+
+jobs:
+  update-labels:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check PR title format
+        id: check_title
+        run: |
+          PR_TITLE="${{ github.event.pull_request.title }}"
+          echo "PR Title: ${PR_TITLE}"
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)